### DISCUSSION TITLE
你想成為狼還是羊？

### DISCUSSION THREAD

**Discussion by 1104030360** [1101-GS4719-C/Python-Community-GS4719-C#1293] [General]
在現在的生活，我們常常在生活中面臨決定要成為影響他人的人(狼)還是順從團體成為被影響的人(羊)，就我而言，我想當狼，因為我覺得成為狼才能在競爭市場上能與人競爭，但如果成為羊，我們就是處於被動的狀態，就算我們是一群羊，還是得由主人帶領，只要主人停下來我們就要停下來，選擇要成為狼還是羊沒有對錯，如果你想成為狼我認為可以在日常生活中訓練做決定的能力，大家可以建構一套自己做決定的系統，盡量不要花太多時間在做決定上，因為時間是金錢，再來是企圖心，我們要清楚了解自己真正想要什麼，並以這個為目標，讓自己有更強大的野心去達到這個目標。

**Comment by CPW0919** [comment]
我想成為披著羊皮的狼，假裝自己是被影響的人，其實默默淺移默化他人。題外話 錯字太多。

**Reply by 1104030360** [reply]
已更正🤪

**Comment by tower20021202** [comment]
如果可以的話，我想成為影響他人的狼，雖然選擇當狼還是羊沒有所謂的好或不好，但我認為現在這社會的優勢方是當狼的人，主動發表自己的意見，負責決定，而不是當一個被動的人

**Comment by Chienyuuu6** [comment]
我也選擇披著羊皮的狼，其實我明白自己內心是能夠成為獨當一面的狼，但礙於環境的種種因素，有些時候假裝自己是隻羊才能夠避免自己在狼群的內鬥中受傷ㄌ

**Comment by YoYooHuang** [comment]
我想當牧羊人。既能保護羊，又不會像狼那樣有影響力。我覺得有時候變成狼的話，羊反而會對你感到反感，這時候不如好好當個牧羊人，好好與這些大群體維持好關係，也與狼有一定的交流，才是我想要的。

**Comment by PaulSu0905** [comment]
我想要可以在不同的情境中採取不同的角色，根據我的目標和價值觀做出適應性的選擇。而這樣要找到一種平衡，能夠充分發揮自己的潛力並對自己的選擇負責。

**Comment by larry40305** [comment]
每個人的道路不同，沒有固定的答案。關鍵在於找到對自己而言最適合的方式，並為之努力。無論你選擇成為狼或羊，重要的是要堅持你的價值觀，並追求你真正想要的生活。

**Comment by aa103967622** [comment]
當羊其實沒有不好，你可以懶洋洋地隨波逐流，不必思考太多，跟著大部分人的想法走基本上不會離正確答案差太遠，也不會錯太多，但是心中可以存留著一隻狼，有時候適時地脫穎而出，可能就是你成功的關鍵一步！

**Comment by ThanatosJun** [comment]
年輕時我可能會當狼，有著滿腔熱血，去追逐我的目標與夢想。但隨著年歲增長，我可能會趨於平淡與安穩，當個披著狼皮的羊，得過且過，後世安穩就好。

**Comment by Alex11392** [comment]
我想成為羊，可以當一種很聰明的羊。
